packages/terra-status-view/src/terra-dev-site/doc/example/StatusViewDocCommon.module.scss

```diff
@@ -0,0 +1,16 @@
+:local {
+  .status-view-1,
+  .status-view-2,
+  .status-view-3 {
```

Comment:
Can we rename to status-view-height, stutus-view-width, status-view-overflow?

Reply:
Done